### PR TITLE
Add alerting scripts

### PR DESCRIPTION
**Pull request by cjakobsen**
Adds scripts to create Azure Monitor action groups and different alert rules.

### CODE REVIEW

File: azure-cli/monitor/alert.utilities.ps1 [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    )
+
+    return az monitor action-group create `
+        --resource-group $resourceGroupName `
+        --name $actionGroupName `
+        --short-name $actionGroupShortName `
+        --tags $tags
+}
+
+function Add-EmailAction{
```
Comment (perkops) [inline azure-cli/monitor/alert.utilities.ps1:56]: Maybe we should add a similar function for adding a webhook - something like.

```
function Add-EmailAction{
    param (
        [Parameter(Mandatory=$true)]
        [ValidateNotNullOrEmpty()]
        [string]
        $resourceGroupName,

        [Parameter(Mandatory=$true)]
        [ValidateNotNullOrEmpty()]
        [string] 
        $actionGroupName,

        [Parameter(Mandatory=$true)]
        [ValidateNotNullOrEmpty()]
        [string] 
        $alertUri
    )

    return az monitor action-group update `
    --resource-group $resourceGroupName `
    --name $actionGroupName `
    --add-action webhook alert_hook $alertUri usecommonalertschema
}
```

File: azure-cli/monitor/alert.utilities.ps1 [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+        [string]
+        $resourceGroupName,
+
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $actionGroupName
+    )
+
+    return az monitor action-group show `
```
Comment (perkops) [inline azure-cli/monitor/alert.utilities.ps1]: could we use list with some JMESPath query instead? show is rather slow..

File: azure-cli/monitor/add_ActionGroup.ps1 [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    # Resource naming section
+    #############################################################################################
+    $resourceGroupName = Get-ResourceGroupName -serviceName $namingConfig.ServiceName -systemName $namingConfig.SystemName -environmentName $environmentConfig.EnvironmentName
+    $actionGroupName = Get-ResourceGroupName -serviceName $namingConfig.ServiceName -systemName $namingConfig.SystemName -environmentName $environmentConfig.EnvironmentName
+    $actionGroupShortName = ($namingConfig.SystemAbbreviation + "-" + $namingConfig.ServiceAbbreviation).ToLower()
+
+    ############################################################################################
+    # Action group section
+    ############################################################################################
+    Write-Host "  Creating action group if not exist" -ForegroundColor DarkYellow
```
Comment (perkops) [inline azure-cli/monitor/add_ActionGroup.ps1]: exists

File: azure-cli/monitor/alert.utilities.ps1 [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+        --resource-group $resourceGroupName `
+        --name $actionGroupName `
+        --query "emailReceivers[?(emailAddress == '$emailAddress')]" `
+        | ConvertFrom-Json
+}
+
+function Get-AllEmailAction{
```
Comment (perkops) [inline azure-cli/monitor/alert.utilities.ps1]: Actions

File: azure-cli/monitor/alert.utilities.ps1 [hunk 5]
```diff
@@ -0,0 +1,259 @@
+. "$PSScriptRoot\..\utilities\deploy.utilities.ps1"
```
Comment (perkops) [inline azure-cli/monitor/alert.utilities.ps1:1]: Maybe we should consider to add some documentation to each method, to describe what the purpose is, and what is being returned - if not obvious.

File: azure-cli/monitor/alert.utilities.ps1 (hunk 1 above)
Comment (cjakobsen) [reply azure-cli/monitor/alert.utilities.ps1:56]: Added.

File: azure-cli/monitor/alert.utilities.ps1 (hunk 2 above)
Comment (cjakobsen) [reply azure-cli/monitor/alert.utilities.ps1]: Changed.

File: azure-cli/monitor/alert.utilities.ps1 (hunk 5 above)
Comment (cjakobsen) [reply azure-cli/monitor/alert.utilities.ps1:1]: Removed a couple of unused functions, the rest should be clear from the names.

**Review by perkops — APPROVED** [review]

**Pull request merged**